<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planted Tank Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>9</y>
      <width>971</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_dry_dosing">
     <attribute name="title">
      <string>Dry Dosing</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>9</y>
        <width>931</width>
        <height>611</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <widget class="QRadioButton" name="drydosing_is_drydosing">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>109</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Dry Dosing</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="drydosing_is_solution">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>40</y>
              <width>109</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Solution</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Container size(ml)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="drydosing_container">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Each dosing</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="drydosing_each_dosing">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Aquarium size(L)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="drydosing_aquarium_size"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Dry fertilizer</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="drydosing_fertilizer"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Dosing method</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QComboBox" name="drydosing_method"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Calculating for</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QComboBox" name="drydosing_element"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Concentration(ppm)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="drydosing_concentration"/>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="drydosing_calculate">
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Element Concentration</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Fertilizer Weight</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="drydosing_element_ppm">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTableWidget" name="drydosing_fertilizer_weight">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="drydosing_add_to_total">
           <property name="text">
            <string>Add to total</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Total Element Concentration</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Total Fertilizer Weight</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QTableWidget" name="drydosing_total_element_ppm">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QTableWidget" name="drydosing_total_fertilizer_weight">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="drydosing_clear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_co2">
     <attribute name="title">
      <string>CO2</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>7</y>
        <width>951</width>
        <height>621</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ptc.qrc">:/images/co2.jpg</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_tank_volume">
     <attribute name="title">
      <string>Tank Volume</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>45</y>
        <width>331</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_18">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>CM</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Length</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="volume_length"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="volume_width"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="volume_high"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Substrate thickness</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="volume_sub_thickness"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Surface to top</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="volume_surface_to_top"/>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="volume_calculate">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>381</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Water volume(L)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="volume_water_volume">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Substrate(L)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="volume_substrate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Surface area(cm2)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="volume_surface">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ptc.qrc"/>
 </resources>
 <connections/>
</ui>
